### DISCUSSION TITLE
Reading Cell value contains 'FALSE' is converted to '0'

### DISCUSSION THREAD

**Discussion by sankar228** [qax-os/excelize#1170] [Q&A]
Hi,

i was trying to use github.com/xuri/excelize/v2 for reading xlsx and convert them to csv. it is working fine for most of my cases.
except for the cell values contain 'FALSE' converted to '0' by excelize instead of treating it as a string value. would you suggest me on this. 
below is the code snippet.

//i have tried with out excelize Options, but the behavior did not change
efile, err := excelize.OpenFile(excelFile, excelize.Options{
		RawCellValue: true,
	})
rows, err := efile.GetRows(sheetName)

for _, row := range rows {
	// logic to read field values and write them to csv file
}

Thanks,
Sankar

**Comment by xuri** [comment]
Thanks for your feedback, could you provide more details and spreadsheet attachments without confidential info?

**Reply by sankar228** [reply]
Thank you for the quick response!!

please find the attached spreadsheet with some sample data.
column "c5" is considered as string, other column values are treated as bool by default.

[test.xlsx](https://github.com/qax-os/excelize/files/8194289/test.xlsx)
.

**Comment by xuri** [comment]
I have fixed this issue on commit [`56aa6b8`](https://github.com/xuri/excelize/commit/56aa6b82637b3210be470a8ebac1fdec2b2a6a30), please try to upgrade master branch code, and this patch will be released on the next version.

**Reply by sankar228** [reply]
Thank you !!

fix is working !!
